server/api/rounds.py

```diff
@@ -991,6 +1005,41 @@ def sample_batches(
                     )
                 )
 
+            # Continue adding batches until the percentage of jurisdiction ballots selected is at
+            # least 2%
+            min_percentage_of_jurisdiction_ballots_selected = 0.02
+
+            def compute_percentage_of_jurisdiction_ballots_selected(
+                selected_batch_ids, num_jurisdiction_ballots
+            ):
+                num_jurisdiction_ballots_selected = sum(
+                    [
+                        batch_id_to_num_ballots[batch_id]
+                        for batch_id in selected_batch_ids
+                    ]
+                )
+                return num_jurisdiction_ballots_selected / num_jurisdiction_ballots
```

Comment:
what is the data type here? We don't run the risk of an integer division forcing this to 0 do we?

Reply:
Good q! In Python, if you want to use integer division, you have to explicitly do so with a double `//`, so we're good! But I'll do some double checking to be sure

Reply:
```
$ python3
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 1 / 3
0.3333333333333333
>>> int(1) / int(3)
0.3333333333333333
>>>
```

^ Another sanity check